sig-contributor-experience/charter.md

```diff
@@ -1,51 +1,45 @@
 # Contributor Experience Special Interest Group Charter
 
-This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
```

Comment:
why did you remove this? this is a part of the charter template.

Reply:
agree with @parispittman's comment